volestipy/setup.py

```diff
@@ -1,68 +1,72 @@
-from os.path import join
 from setuptools import setup
+from setuptools import Extension
 from Cython.Build import cythonize
-from setuptools.extension import Extension
+from os.path import join
 import numpy
 
 
-version = "0.2.0"
+version = "0.3.0"
 license='LGPL3',
 packages = ["volestipy"]
 description="volestipy: wrapper for the VolEsti library to sample from convex sets and compute volume."
-author = "Pedro Zuidberg Dos Martires"
-author_email="pedro.zuidbergdosmartires@cs.kuleuven.be"
+author = "Haris Zafeiropoulos"
```

Comment:
please do not remove the original author, just add your name

Reply:
Terribly sorry for this. I did so in the first place to distinguish the two scripts.